central/reports/common/query_builder.go

```diff
@@ -1,47 +1,64 @@
 package common
 
 import (
+	"context"
 	"fmt"
 	"strings"
 	"time"
 
+	collectionDataStore "github.com/stackrox/rox/central/resourcecollection/datastore"
 	v1 "github.com/stackrox/rox/generated/api/v1"
 	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/features"
 	"github.com/stackrox/rox/pkg/sac/effectiveaccessscope"
 	"github.com/stackrox/rox/pkg/search"
 )
 
 // ReportQuery encapsulates the cve specific fields query, and the resource scope
 // queries to be used in a report generation run
 type ReportQuery struct {
-	CveFieldsQuery string
-	ScopeQueries   []string
+	CveFieldsQuery   string
+	ScopeQueries     []string
```

Comment:
Comments to explain why both scope queries and deployment query exists would be nice